kornia/contrib/sam/architecture/__init__.py

```diff
@@ -0,0 +1,4 @@
+from kornia.contrib.sam.architecture.image_encoder import ImageEncoderViT
+from kornia.contrib.sam.architecture.mask_decoder import MaskDecoder
+from kornia.contrib.sam.architecture.prompt_encoder import PromptEncoder
+from kornia.contrib.sam.architecture.transformer import TwoWayTransformer
```

Comment:
i wouldn't expose those -- in the internal implementations we can just import specifying the full path. That will help to alleviate potential circular dependencies